docs/components/modals/index.html

```diff
@@ -42,19 +40,19 @@ <h3>Demo Modal Header</h3>
           </hx-disclosure>
 
           <hx-modal id="demo-modal">
-            <hx-modalhead>
+            <header>
               <h3>Demo Modal Header</h3>
-            </hx-modalhead>
-            <hx-modalbody>
+            </header>
+            <hx-div>
               <p>
                 This is the body of a demo modal. No further actions can happen
                 on the page until this modal is closed.
               </p>
-            </hx-modalbody>
-            <hx-modalfoot>
+            </hx-div>
+            <footer>
               <button class="hxBtn hxPrimary">Confirm</button>
               <button class="hxBtn">Cancel</button>
-            </hx-modalfoot>
+            </footer>
           </hx-modal>
         {% endcode %}
       </footer>
```

Comment:
Add a "Scrolling Modal" section to demonstrate how to use `<hx-div scroll="vertical">`.